ui/operator_ui/main.py

```diff
@@ -523,6 +525,7 @@ def get_postgresqls():
             'namespaced_name': namespace + '/' + name,
             'full_name': namespace + '/' + name + ('/' + uid if uid else ''),
             'status': status,
+            'num_elb': spec.get('enableMasterLoadBalancer', 0) + spec.get('enableReplicaLoadBalancer', 0),
```

Comment:
maybe you can also add support for `enableMasterPoolerLoadBalancer` and `enableReplicaPoolerLoadBalancer` so that they are included here? :)

Reply:
is the cost the same for the pooler load balancer?

Reply:
Yes